<Type Name="AddTaskCollectionTerminatedException" FullName="Microsoft.Azure.Batch.AddTaskCollectionTerminatedException">
  <TypeSignature Language="C#" Value="public class AddTaskCollectionTerminatedException : Microsoft.Azure.Batch.BatchClientException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi beforefieldinit AddTaskCollectionTerminatedException extends Microsoft.Azure.Batch.BatchClientException" />
  <TypeSignature Language="DocId" Value="T:Microsoft.Azure.Batch.AddTaskCollectionTerminatedException" />
  <TypeSignature Language="VB.NET" Value="Public Class AddTaskCollectionTerminatedException&#xA;Inherits BatchClientException" />
  <TypeSignature Language="F#" Value="type AddTaskCollectionTerminatedException = class&#xA;    inherit BatchClientException" />
  <AssemblyInfo>
    <AssemblyName>Microsoft.Azure.Batch</AssemblyName>
    <AssemblyVersion>7.1.0.0</AssemblyVersion>
    <AssemblyVersion>8.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>Microsoft.Azure.Batch.BatchClientException</BaseTypeName>
  </Base>
  <Interfaces />
  <Docs>
    <summary>
            <span data-ttu-id="a0a97-101">Die Ausnahme, die ausgelöst wird, wenn der Vorgang AddTaskCollection beendet wird.</span><span class="sxs-lookup"><span data-stu-id="a0a97-101">The exception that is thrown when the AddTaskCollection operation is terminated.</span></span>
            </summary>
    <remarks>To be added.</remarks>
  </Docs>
  <Members>
    <Member MemberName="AddTaskResult">
      <MemberSignature Language="C#" Value="public Microsoft.Azure.Batch.AddTaskResult AddTaskResult { get; }" />
      <MemberSignature Language="ILAsm" Value=".property instance class Microsoft.Azure.Batch.AddTaskResult AddTaskResult" />
      <MemberSignature Language="DocId" Value="P:Microsoft.Azure.Batch.AddTaskCollectionTerminatedException.AddTaskResult" />
      <MemberSignature Language="VB.NET" Value="Public ReadOnly Property AddTaskResult As AddTaskResult" />
      <MemberSignature Language="F#" Value="member this.AddTaskResult : Microsoft.Azure.Batch.AddTaskResult" Usage="Microsoft.Azure.Batch.AddTaskCollectionTerminatedException.AddTaskResult" />
      <MemberType>Property</MemberType>
      <AssemblyInfo>
        <AssemblyName>Microsoft.Azure.Batch</AssemblyName>
        <AssemblyVersion>7.1.0.0</AssemblyVersion>
        <AssemblyVersion>8.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <ReturnValue>
        <ReturnType>Microsoft.Azure.Batch.AddTaskResult</ReturnType>
      </ReturnValue>
      <Docs>
        <summary>
            <span data-ttu-id="a0a97-102">Ruft die <see cref="P:Microsoft.Azure.Batch.AddTaskCollectionTerminatedException.AddTaskResult" /> für die Aufgabe, die Ausnahme verursacht hat.</span><span class="sxs-lookup"><span data-stu-id="a0a97-102">Gets the <see cref="P:Microsoft.Azure.Batch.AddTaskCollectionTerminatedException.AddTaskResult" /> for the task which caused the exception.</span></span>
            </summary>
        <value>To be added.</value>
        <remarks>
            <span data-ttu-id="a0a97-103">Mehr als eine Aufgabe ist möglicherweise fehlgeschlagen.</span><span class="sxs-lookup"><span data-stu-id="a0a97-103">More than one task may have failed.</span></span> <span data-ttu-id="a0a97-104">Um die vollständige Liste angezeigt wird, verwenden Sie eine <see cref="T:Microsoft.Azure.Batch.AddTaskCollectionResultHandler" />.</span><span class="sxs-lookup"><span data-stu-id="a0a97-104">In order to see the full list, use an <see cref="T:Microsoft.Azure.Batch.AddTaskCollectionResultHandler" />.</span></span>
            </remarks>
      </Docs>
    </Member>
  </Members>
</Type>